R/pkg/R/generics.R

```diff
@@ -465,10 +477,14 @@ setGeneric("dapply", function(x, func, schema) { standardGeneric("dapply") })
 #' @export
 setGeneric("dapplyCollect", function(x, func) { standardGeneric("dapplyCollect") })
 
+#' @param x a SparkDataFrame or GroupedData.
```

Comment:
I think there is also one for [SparkDataFrame](https://github.com/apache/spark/pull/14558/files#diff-508641a8bd6c6b59f3e77c80cdcfa6a9L1401)